README.md

```diff
@@ -8,4 +8,6 @@ Start a postgres database:
 
 Link it to netdisco:
 
-`docker run -d --name=netdisco -e NETDISCO_WR_COMMUNITY="private" --link pg-netdisco:db sheeprine/docker-netdisco`
+`docker run -d --name=netdisco -e NETDISCO_WR_COMMUNITY="private" --link pg-netdisco:db -p 5000:5000 sheeprine/docker-netdisco`
+
+Connect using your browser to http://<IP-of-docker-host>:5000/
```

Comment:
Since we are here, maybe hinting at the default user/pass might be a good thing.